renovate.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["github>ministryofjustice/hmpps-renovate-config:node"],
```

Comment:
It's worth folks also having a look through
- https://github.com/ministryofjustice/hmpps-renovate-config/blob/main/base.json
- https://github.com/ministryofjustice/hmpps-renovate-config/blob/main/node.json